content/20.treatments.md

```diff
@@ -302,8 +302,19 @@ All of these factors could potentially influence treatment outcome.
 Furthermore, the authors acknowledge that the effect of the drugs might be different in females and pediatric subjects, since these subjects were not the part of the study.
 
 **Summary.**
-In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19.
-Multiple clinical studies have already been carried out to assess this possibility.
+In vitro evidence shows HCQ may be an effective therapeutic against SARS-CoV-2 and COVID-19. 
+It was recently discussed [@doi:10.1093/jac/dkaa114] that since EC90 of CQ in Vero E6 cells (6.90 μM) could be reached and well endured in rheumatoid arthritis patients, it could also be administered to COVID-19 patients.
+Additionally, it was proven to be effective against HIV [@doi:10.1016/0149-2918(95)80039-5] and chronic Hepatitis C infection [@doi:10.1002/jmv.24575].
+Therefore, initially it was thought that it would be effective towards SARS-COV-2.
+Further, HCQ was hypothesized to be safer as compared to CQ [@doi:10.1093/jac/dkaa114].
+The hypotheses were that HCQ could alleviate the development of COVID-19 through mitigation of CD154 in T cells,  could have the same effect as an antiviral like CQ pre- and post-infection, is cheaper, and has less side effects than CQ. 
+HCQ is safer for pregnant women with autoimmune disease. 
+This could possibly be the reason behind HCQ being predominantly used in clinical trial as opposed to CQ.
+Furthermore, a separate study [@doi:10.1101/2020.04.08.20054551] investigated the safety associated with the use of this drug in COVID-19. 
+The study involved 900,000 cases with HCQ and 300,000 cases with HCQ+AZ. 
+The results showed that short term use of the drug was associated with no risk while the long-term use was associated with 65% increase in cardiovascular morbidity. 
```

Comment:
```suggestion
The results showed that short-term use of the drug was associated with no risk, but long-term use was associated with a 65% increase in cardiovascular morbidity. 
```
I'm not sure I totally understand what they're comparing to, since all of the 1,200,000 patients described in the sentence above would have received HCQ it sounds like?

Reply:
We can strike out " but long-term use was associated with a 65% increase in cardiovascular morbidity". This is with RA patients.

Reply:
@soumitagh is above what you meant?